src/Microsoft.Extensions.Configuration.Abstractions/LoadExceptionContext.cs

```diff
@@ -6,14 +6,14 @@
 namespace Microsoft.Extensions.Configuration
 {
     /// <summary>
-    /// Contains information about a file load exception.
+    /// Contains information about a load exception.
     /// </summary>
-    public class FileLoadExceptionContext
+    public class LoadExceptionContext
```

Comment:
I think it could be a good idea to give this more of a unique name independent of the namespace, e.g. `ConfigurationLoadExceptionContext`